### PR TITLE
Decimal is a number

### PR DESCRIPTION
**Pull request by kooky**
Lots of fields come from MySQL as type Decimal. And they are a number.

### CODE REVIEW

File: CHANGES [hunk 1]
```diff
@@ -21,6 +21,7 @@ Version 2.8
 - Implemented a block ``set`` tag.
 - Default cache size was incrased to 400 from a low 50.
 - Fixed ``is number`` test to accept long integers in all Python versions.
+- ``is number`` to accept Decimal as a number.
```
Comment (untitaker) [inline CHANGES]: A word is missing here.

**Comment by untitaker** [comment]
Could you use `git rebase -i master` to produce just one commit?

**Comment by kooky** [comment]
`git rebase -i master` has made me lose the will to live.

**Comment by untitaker** [comment]
Ok, applied as https://github.com/mitsuhiko/jinja2/commit/cdeaa33d0f049641b2b0af108593449842a8d238. Thanks!

**Pull request closed without merging**